src/CMakeLists.txt

```diff
@@ -38,7 +38,13 @@ set_property(TARGET nvcomp PROPERTY CUDA_ARCHITECTURES ${GPU_ARCHS})
 target_compile_options(nvcomp PRIVATE
     $<$<COMPILE_LANGUAGE:CUDA>:--expt-extended-lambda>)
 target_include_directories(nvcomp PUBLIC
-    "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../include>")
+    "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../include>"
+    )
+if(UNIX)
+target_include_directories(nvcomp PUBLIC
+  "$<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>"
```

Comment:
Can you confirm that this is your required change? The rest of the cmake changes are optional, right?

Reply:
@eschmidt-nvidia It's been a while since I wrote this patch so my memory is a bit foggy.  As I recall, this isn't the only required change.  Specifically, including GNUInstallDirs where it was previously included meant that some of the variables were not defined in the right place

The CMAKE_EXPORT_COMPILE_COMMANDS is optional, but enables some tooling.